dev-util/envman/envman-2.2.8.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit golang-build
+
+EGO_ON="github.com/bitrise-io"
+EGO_PN="${EGO_ON}/${PN}"
+
+DESCRIPTION="Environment variable manager for Bitrise CLI"
+HOMEPAGE="https://www.bitrise.io/cli"
+SRC_URI="https://${EGO_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+RESTRICT="test"
```

Comment:
Why exactly is this restricted?

Reply:
https://github.com/gentoo/gentoo/pull/13245#discussion_r500898006